drake/systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m

```diff
@@ -406,7 +406,7 @@
         f_friction = f_friction + min(1,max(-1,v./coulomb_window)).*coulomb_friction;
         if compute_gradient
           ind = find(abs(v)<coulomb_window');
-          dind = sign(v(ind))./coulomb_window(ind)' .* coulomb_friction(ind)';
+          dind = coulomb_friction(ind)' ./coulomb_window(ind)';
```

Comment:
yeah, this was that one place where the gradients turned out to be wrong in both C++ and Matlab.